atlantis-ci/ecs_service.tf

```
@@ -25,15 +25,15 @@ module "atlantis" {
       },
       {
         name  = "ATLANTIS_GH_APP_ID"
-        value = "${var.github_app_id}"
+        value = "var.github_app_id"

```

This will turn string substitution into a simple string.

Should be 
```suggestion
        value = var.github_app_id

```

---

done